src/nanoemoji/color_glyph.py

```diff
@@ -211,15 +237,15 @@ def shape_cache_key(self):
         return (self.path, self.reuses)
 
 
-def _paint(debug_hint, upem, picosvg, shape):
+def _paint(debug_hint: str, config: FontConfig, picosvg: SVG, shape: SVGPath):
```

Comment:
optional: annotate the return value as well (-> Paint), since you annotated all the parameters